nuxeo-admin-console-web/angular-app/src/app/features/sub-features/generic-multi-feature-layout/components/folder-tab/folder-tab.component.ts

```diff
@@ -73,6 +73,7 @@ export class FolderTabComponent implements OnInit, OnDestroy {
   templateLabels: labelsList = {} as labelsList;
   requestQuery = "";
   activeFeature: FeaturesKey = {} as FeaturesKey;
+  featuresRequiringOnlyId = ["elasticsearch-reindex"];
```

Comment:
Can add this in constants instead of inside a component, so that any other upcoming feature can be added as well

Reply:
changes  done